src/main/archetype/ui.tests/docker-compose-wdio-chrome.yaml

```diff
@@ -34,6 +34,7 @@ services:
     volumes:
       - ./target/reports:/tmp/reports
       - 'shared:/tmp/shared'
+      - 'shared:/usr/src/app/assets'
```

Comment:
I assume this is to compare the test image resource during the run, I don't see the code which copies the source image to this directory ?

Reply:
This is the configuration for docker volume, what it does is the volume ui.tests will have files from the folder specified here.
```
      - 'shared:/usr/src/app/assets'
```
The docker will map this folder from container to volume, the volume will have persistent data.
ie the docker handles the copying.